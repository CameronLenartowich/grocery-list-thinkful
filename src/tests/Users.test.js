import React from 'react';
import ReactDOM from 'react-dom';
import UsersTest from '../components/Users';
import { exportAllDeclaration } from '@babel/types';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';
Enzyme.configure({ adapter: new Adapter() });
import * as firebase from 'firebase';
const firebaseConfig = {
    apiKey: "AIzaSyDiMKeTwz7LgFfpPRNVc3dDQYH_A4e48Uo",
    authDomain: "grocery-list-1ed7b.firebaseapp.com",
    databaseURL: "https://grocery-list-1ed7b.firebaseio.com",
    projectId: "grocery-list-1ed7b",
    storageBucket: "grocery-list-1ed7b.appspot.com",
    messagingSenderId: "833209511385",
    appId: "1:833209511385:web:31f159d5098c2a96"
  };
  firebase.initializeApp(firebaseConfig);

const Users = (props) => 
    <UsersTest
        firebase = {firebase}
        user={"guest"}
    />;

it('render correctly Users component', () => {  
    const component = renderer.create(<Users />).toJSON();
    expect(component).toMatchSnapshot();
});