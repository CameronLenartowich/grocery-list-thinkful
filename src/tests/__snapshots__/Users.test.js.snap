// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly Users component 1`] = `
<span>
  <span>
    <p>
      Signed in as 
      guest
    </p>
    <button
      onClick={[Function]}
    >
      Sign Out
    </button>
  </span>
</span>
`;
